src/build.cake

```diff
@@ -46,7 +46,7 @@ var nodeBinaryUrl = "https://github.com/sourcegraph/node-binaries/raw/main/v20.1
 var nodeArmBinaryUrl = "https://github.com/sourcegraph/node-binaries/raw/main/v20.12.2/node-win-arm64.exe";
 
 // The latest tag of the stable release from the cody repository https://github.com/sourcegraph/cody/tags
-var codyStableReleaseTag = "vscode-v1.38.1";
+var codyStableReleaseTag = "vscode-v1.40.0";

```

Comment:
```suggestion
var codyStableReleaseTag = "vscode-v1.40.0";
```

Update this value to the commit hash or version tag used in the cody main repo if needed